MusicStore.Web/Controllers/MusicStoreController.cs

```diff
@@ -25,19 +26,28 @@ public ActionResult DisplayAvailableMusicForLoggedUser()
         [HttpPost]
         public ActionResult BuyMusic(int userId, int songId)
         {
-            if(userId < 0 || songId < 0)
+            if (userId < 0 || songId < 0)
             {
-                return new HttpStatusCodeResult(HttpStatusCode.Unauthorized);
+                return new HttpStatusCodeResult(HttpStatusCode.BadRequest, "userId or songId is null");
             }
             try
             {
-                bool resultOfBuy = _musicStoreService.BuySong(songId, userId);
-                ViewBag.OperationResult = resultOfBuy ? "Покупка совершена успешно" : "Покупка не совершена успешно";
+                var resultOfBuy = _musicStoreService.BuySong(songId, userId);
```

Comment:
For future:
You can not identify type of variable by "Just Looking and **this** code". In future, specify type of variable